mismi-autoscaling-core/mismi-autoscaling-core.cabal

```diff
@@ -0,0 +1,42 @@
+name:                  ambiata-mismi-autoscaling-core
+version:               0.0.1
+license:               Apache-2.0
+license-file:          LICENSE
+author:                Ambiata <info@ambiata.com>
+maintainer:            Ambiata <info@ambiata.com>
+copyright:             (c) 2015 Ambiata
+synopsis:              AWS library
+category:              AWS
+cabal-version:         >= 1.8
+build-type:            Simple
+description:           mismi-autoscaling-core.
+
+library
+  build-depends:
+                       base                            >= 3          && < 6
+                     , ambiata-p
```

Comment:
Any reason for the `6` upper bound on base? Ghc 8.0 shipped with `4.9` and 8.2 is likely to ship with `4.10`.